packages/@angular/cli/tasks/npm-install.ts

```diff
@@ -6,17 +6,18 @@ import {exec} from 'child_process';
 export default Task.extend({
   run: function() {
     const ui = this.ui;
+    const packageManager = this.packageManager;
 
     return new Promise(function(resolve, reject) {
-      ui.writeLine(chalk.green('Installing packages for tooling via npm.'));
-      exec('npm install',
+      ui.writeLine(chalk.green(`Installing packages for tooling via ${packageManager}.`));
+      exec(`${packageManager} install`,
```

Comment:
No need to change now, but if we do add support for another package manager that may not use `install` as the command, this code will need updated.